app/interactors/tasks/create.rb

```diff
@@ -2,7 +2,13 @@ module Tasks
   class Create
     include Interactor::Organizer
 
+    delegate :project, :task, to: :context
```

Comment:
Here task will be taken from save interactor, you don't really needed to pass it (comment https://github.com/fs/task_tracker_itis_2023/pull/100/files#r1385405269)